src/theme/DocItem/index.js

```diff
@@ -0,0 +1,223 @@
+                            </em>
+                        )}
+                    </div>
+                )}
+            </div>
+        </div>
+    );
+}
+
+function DocItem(props) {
+    const { siteConfig = {} } = useDocusaurusContext();
+    const { url: siteUrl, title: siteTitle } = siteConfig;
+    const { content: DocContent } = props;
+    const { metadata } = DocContent;
+    const { description, title, permalink, version } = metadata;
+    const {
+        frontMatter: { image: metaImage, keywords, hide_title: hideTitle, hide_table_of_contents: hideTableOfContents },
+    } = DocContent;
+
+    const metaTitle = title ? `${title} | ${siteTitle}` : siteTitle;
+    let metaImageUrl = siteUrl + useBaseUrl(metaImage);
+    if (!isInternalUrl(metaImage)) {
+        metaImageUrl = metaImage;
+    }
+
+    return (
+        <>
+            <Head>
+                <title>{metaTitle}</title>
+                <meta property="og:title" content={metaTitle} />
+                {description && <meta name="description" content={description} />}
+                {description && <meta property="og:description" content={description} />}
+                {keywords && keywords.length && <meta name="keywords" content={keywords.join(",")} />}
+                {metaImage && <meta property="og:image" content={metaImageUrl} />}
+                {metaImage && <meta property="twitter:image" content={metaImageUrl} />}
+                {metaImage && <meta name="twitter:image:alt" content={`Image for ${title}`} />}
+                {permalink && <meta property="og:url" content={siteUrl + permalink} />}
+                {permalink && <link rel="canonical" href={siteUrl + permalink} />}
+            </Head>
+            <div className={classnames("container padding-vert--lg", styles.docItemWrapper)}>
+                <div className="row">
+                    <div
+                        className={classnames("col", {
+                            [styles.docItemCol]: !hideTableOfContents,
+                        })}
+                    >
+                        <div className={styles.docItemContainer}>
+                            <article>
+                                {version && (
+                                    <div>
+                                        <span className="badge badge--secondary">Version: {version}</span>
+                                    </div>
+                                )}
+                                {!hideTitle && (
+                                    <header>
+                                        <h1 className={styles.docTitle}>{title}</h1>
+                                    </header>
+                                )}
+                                <div className="markdown">
+                                    <DocContent />
+                                </div>
+                            </article>
+                            <DocMetadata content={DocContent} />
```

Comment:
Oh I guess we can move this up to the top here (see previous comment about moving date up to the top of the article)